docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -157,7 +157,7 @@ If you deploy Camunda 8 (and related infrastructure) with permissive SCCs out of
 
 ## Ingress Configuration
 
-Before exposing services outside the cluster, we need an ingress component. Here's how you can configure it:
+Before exposing services outside the cluster, we need an Ingress component. Here's how you can configure it:
```

Comment:
I don't think we're consistent with ingress/Ingress casing across our docs. @conceptualshark WDYT? Do you think this warrants an issue for the backlog? Would you like to take a look at this across the SM docs?

Reply:
👍 I can spin something off to go through this. I think the capitalization does a lot of work here.